api/graphql/schema.graphql

```diff
@@ -959,7 +959,9 @@ type Query {
   poolAdvertisements: [PoolAdvertisement]!
     @all(model: "Pool")
     @can(ability: "viewAny", model: "Pool")
-  publishedPoolAdvertisements: [PoolAdvertisement!]!
+  publishedPoolAdvertisements(
+    closingDate: DateTime @where(key: "closing_date", operator: ">")
```

Comment:
I think it may benefit us to make this input a little clearer or allowing the user to also select the operator. I'm thinking something like `closingAfter`?

Reply:
Renamed in https://github.com/GCTC-NTGC/gc-digital-talent/pull/6479/commits/00d7f8944c198f4d91b716f82afd68d1acf8aea0
Given this query is policy-wise, completely open, I don't see it needing stuff like scoping for only closed ever.